tools/number.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
suggest to let the bash command seq do the work

Reply:
Strictly speaking `seq` doesn't have the same behavior as `NUMBER` on negative values of incr (`seq` does the useful thing and `NUMBER` stops after 1 step). But I guess that means no one is using the behavior anyway...

Reply:
Not sure if you are putting this comment on the shebang line to point it out, but as an FYI, this one uses `bash` instead of `sh` because it needs the `[[ "$1" == ?(-)+([0-9]) ]]` command which doesn't work in `sh`. Maybe you have a better idea how to test for numerals in posix compliant mode?

Reply:
check out here:
https://stackoverflow.com/questions/4137262/is-there-an-easy-way-to-determine-if-user-input-is-an-integer-in-bash

Reply:
Why are we concerned with a Unix shell, but not bash?  translate.sh explicitly refers to bash.  Lets downsize in a different  PR and assume bash for now.

Reply:
Works for me. Writing portable shell scripts is a headache.